src/app/SimpleSubscriptionResumptionStorage.h

```diff
@@ -59,6 +59,8 @@ class SimpleSubscriptionResumptionStorage : public SubscriptionResumptionStorage
 private:
     CHIP_ERROR SaveIndex(const SubscriptionIndex & index);
     CHIP_ERROR SaveSubscriptions(const ScopedNodeId & node, const SubscriptionList & subscriptions);
+    CHIP_ERROR LoadIndex(SubscriptionIndex & index, size_t allocateExtraSpace);
+    CHIP_ERROR FindByScopedNodeId(ScopedNodeId node, SubscriptionList & subscriptions, size_t allocateExtraSpace);
```

Comment:
The meaning of these arguments would be good to document. They are not immediately obvious. The expectations about the API (who owns the memory, who manages it) need to be clearlys tated, since the `subscriptions` object and `SubscriptionIndex` object are dynamically allocated.

Reply:
The storage API has changed to the iterator pattern and should be much more intuitive now.